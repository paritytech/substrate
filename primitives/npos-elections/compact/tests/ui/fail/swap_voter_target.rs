use sp_npos_elections_compact::generate_solution_type;

generate_solution_type!(pub struct TestSolution::<
	TargetIndex = u16,
	VoterIndex = u8,
	Accuracy = Perbill,
>(8));

fn main() {}
