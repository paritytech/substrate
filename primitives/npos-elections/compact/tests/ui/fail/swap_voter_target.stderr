error: Expected `VoterIndex`
 --> $DIR/swap_voter_target.rs:4:2
  |
4 |     TargetIndex = u16,
  |     ^^^^^^^^^^^
